main.js

```diff
@@ -0,0 +1,111 @@
+const express = require('express')
+// Get the client
+const bodyParser = require('body-parser');
+const { getLastIPAddress, insertLastIPAddress } = require('./db');
+const { getDistanceFromLatLonInKm, calculateRiskScore } = require('./utils');
+const Reader = require('@maxmind/geoip2-node').Reader;
+// Typescript:
+// import { Reader } from '@maxmind/geoip2-node';
+
+const options = {
+  // you can use options like `cache` or `watchForUpdates`
+};
+
+function getLatLong(IPAddress) {
+  return new Promise((resolve, reject) => {
+    Reader.open('./GeoLite2-City.mmdb').then(reader => {
+      const response = reader.city(IPAddress);
+      resolve({
+        latitude: response.location.latitude,
+        longitude: response.location.longitude
+      })
+    }).catch((error) => {
+      reject(error);
+    });
+  })
+}
+
+
+const app = express()
+const port = 3000
+
+app.use( bodyParser.json() );       // to support JSON-encoded bodies
+app.use(bodyParser.urlencoded({     // to support URL-encoded bodies
+  extended: true
+})); 
+// Get the client
+
+app.get('/', (req, res) => {
+  console.log("request recieved ", req);
+  res.send('Hello World!')
+})
+
+app.post('/check-for-anomaly', async (req, res) => {
+  console.log('receiving data ...');
+  console.log('body is ', req.body);
+  const { userId, newIpAddress } = req.body;
+
+  try {
+    Promise.all([
```

Comment:
why not use async await? Please convert the code to use async / await

Reply:
This has been changed to use async await. Async await and Promise.all serve different puproses. I wanted to trigger two requests simultaneosly here so that time can be saved. async await will make both requests sequential. 

With the new code changes on LocationDB, the request is not async anymore. So, removed Promise.All.